pkgs/development/tools/xcbuild/platform.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, sdk, writeText, platformName, xcbuild }:
+
+let
+
+  Info = {
+    CFBundleIdentifier = platformName;
+    Type = "Platform";
+  };
```

Comment:
It's probably a good idea to set a `Name = "macosx";` or something here. Some code in xcbuild may rely on that being non-empty, IIRC.

(I noticed because a few of the other commits are referencing a directory `Release-`, which is special cased to just `Release` when the platform name is `macosx`.)